src/scss/_variables.scss

```diff
@@ -1,5 +1,5 @@
 $n-alert-banner-is-silent: true !default;
 
 $_n-alert-banner-themes: ('error', 'neutral', 'success', 'marketing');
-$_n-alert-banner-spacing: oTypographySpacingSize($units: 10) !default;
+$_n-alert-banner-spacing: 40px !default;
```

Comment:
I faffed around with o-spacing for ages, then realised I was wasting valuable time and that o-spacing doesn't have an exact match. I didn't want to make the design call so I've hard-coded it for now.